interface-definitions/include/dhcp/ddns-settings.xml.i

```diff
@@ -0,0 +1,172 @@
+<!-- include start from dhcp/ddns-settings.xml.i -->
+<leafNode name="send-updates">
+    <properties>
+        <help>Enable or disable updates for this scope</help>
```

Comment:
As far as I can see, these options are all [boolean](https://github.com/isc-projects/kea/blob/master/src/bin/dhcp4/dhcp4_parser.yy#L698-L702). Is there really a reason not to make it a valueless node `send-updates`?